testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode/multinode-master-jvm.config

```diff
@@ -18,3 +18,4 @@
 -DHADOOP_USER_NAME=hive
 -Duser.timezone=Asia/Kathmandu
 -XX:ErrorFile=/docker/logs/product-tests-presto-jvm-error-file.log
+-javaagent:/docker/jacoco-agent.jar=output=tcpserver,address=*,port=57026,includes=io.trino.*
```

Comment:
Yes, it's needed for getting coverage on multinode PT envs.

The `-javaagent` option added to `run-launcher` only creates coverage for the laucher which is running on host. 
All containers run separate JVMs which also need to generate coverage.

Coverage is typically written by a JVM shutdown handler, but in the case of Trino containers it turns out JVM shutdown handlers aren't called in PT, the PT framework simply kills the entire container. So I chose to use `server` mode for the coverage agent and polling for latest coverage data from the health check script which is also running in the same container. The `57026` is just a random port number. I will use one from a 1024-32768 range instead .

Reply:
I totally misread this, sorry. 

- `run-launcher` should be just w wrapper script.
- it already adds support for `LAUNCHER_OPTS`
   - this should be renamed to LAUNCHER_JAVA_OPTS, since it's options passed to Launcher's java, and not passed as launcher options
   - the variable can be used to activate the agent on CI

Anyway, what does the coverage of the PTL give us, actually?

>  I will use one from a 1024-32768 range instead .

is it possible to bind to 0, and then inquire what port the thing was bound to?